packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Client-based rate limits
+excerpt: >-
+  Understand what client-based rate limits at Okta and learn how to design for efficient use of resources
+---
+
+# Client-based rate limits
+
+Client-based rate limiting for the `/authorize` endpoint provides granular isolation between requests made to the `/authorize` endpoint using a combination of the Client ID, user's IP address, and Okta device identifier. This framework isolates rogue OAuth clients and bad actors, thereby ensuring valid users and applications don't run into rate limit violations.
+
+The [`/authorize` endpoint](/docs/reference/api/oidc/#authorize) is the starting point for browser-based OpenID Connect flows such as the [Implicit flow](/docs/concepts/oauth-openid/#implicit-flow) or the [Authorization Code flow](/docs/concepts/oauth-openid/#authorization-code-flow). A request to this endpoint authenticates the user and either returns an authorization grant or tokens to the client application as part of the callback response.
+
+Currently, client-based rate limits apply to the OAuth `/authorize` endpoint on both the Okta Org Authorization server and any Custom Authorization Server. All Custom Authorization Servers share a rate limit. The Org Authorization Server has a separate rate limit.
```

Comment:
Should we link them to the endpoint rate limits page from here: To understand the rate limits ...refer here?